### DISCUSSION TITLE
tree组件代码中有关于搜索功能的实现，怎么使用它？

### DISCUSSION THREAD

**Discussion by cubita-io** [layui/layui#1545] [问答]
### 议题条件

- [X] 我确认已查看官方使用文档：https://layui.dev ，但没有找到相关解决方案。
- [X] 我确认已在 [Issues](https://github.com/layui/layui/issues?q=is%3Aissue ) 中搜索过类似的问题，但没有找到相关解决方案。
- [X] 我已仔细阅读： 🍀[ Layui Issue 贡献指南](https://github.com/layui/layui/blob/main/CONTRIBUTING.md)

### 议题类型

功能请求

### 使用版本

2.9.2

### 问题描述

tree组件代码中有关于搜索功能的实现，怎么使用它？
![image](https://github.com/layui/layui/assets/59306969/113c2cda-8222-492b-9d9c-702d8a0fba8b)

### 业务代码

```auto
tree组件代码中有关于搜索功能的实现，怎么使用它？
```

### 浏览器

谷歌

### 友好承诺

- [X] 我承诺将本着相互尊重、理解和友善的态度进行交流，共同维护 Layui 良好的社区氛围。

**Comment by sentsim** [comment]
tree 的搜索功能并没有对外开启，也没有在文档提及，这类功能一般不建议使用。
